docs/symptom-survey/coding.md

```diff
@@ -475,6 +475,7 @@ new items were meant to capture reasons for vaccine hesitancy among respondents.
 * C14a is a revision of item C14, changed from "the past 5 days" to "the past
   7 days" to be consistent with other items on the COVID Symptom Survey.
   C14a replaces C14.
+*  C17a is a revision of item C17, which asked respondents if they have had a flu vaccination since June 2020, changed to July 1, 2020.
```

Comment:
```suggestion
*  C17a is a revision of item C17, which asked respondents if they have had a flu vaccination since June 2020. C17a changed the date to July 1, 2020 and simplified the wording.
```
grammatical issues and expansion